src/guide/jbs-jdk-bug-system.md

```diff
@@ -163,11 +206,10 @@ Now that the issue is in the right component and has the basic information, the
 1. Ensure the priority is correct.
    * An approach that has been used for getting a consistent view of priority is to consider three aspects of the issue: **Impact** of the issue; **Likelihood** of it occurring; and, whether there is a **Workaround**. The higher the impact and likelihood the higher the priority; then, having a workaround reduces that priority - but mostly where the impact and likelihood aren't that severe.
 1. Ensure the [Affects Version/s]{.jbs-field} field is correct (within reason).
-   * This may involve reproducing the bug, if doing so is fast and easy.
-   * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
-   * The [Affects Version/s]{.jbs-field} isn't meant to be an exhaustive list of releases the issue is relevant to - it should initially be set to the release the issue was reproduced or identified on, and by implication it will be relevant on all releases past that point (see the [(Rel)[-na]{.jbs-label}](#rel-na) label). If it's later found to be applicable to an earlier release family then adding that earlier release is encouraged if the issue needs to be fixed in that release.
-   * Do not add additional release values to [Affects Version/s]{.jbs-field} for the same release family. E.g. If there is the value [11.0.2]{.jbs-value}, don't add [11.0.5]{.jbs-value}, [11.0.7]{.jbs-value} etc. Adding an additional value for a separate release family where it's still reproducible, e.g. [12]{.jbs-value}, isn't necessary but ok, especially if there's been a few releases since the latest version noted. E.g. [Affects Version/s]{.jbs-field} is [8]{.jbs-value} but its still relevant to the latest mainline release.
-   * For enhancements the [Affects Version]{.jbs-field} should be empty unless you feel that it is only relevant to a particular release family, and should not go into a future mainline release.
+  * This may involve reproducing the bug, if doing so is fast and easy.
+  * In addition to the version where the bug was found, take special care to also investigate if the bug affects mainline.
+    * See [Indicating what releases an issue is applicable to](#indicating-what-releases-an-issue-is-applicable-to) for more details.
+  * For enhancements the [Affects Version]{.jbs-field} should be empty unless you feel that it is only relevant to a particular release family, and should not go into a future mainline release.
```

Comment:
It looks these should be indented to be rendered as the subitems of “Ensure the Affects Version/s field is correct (within reason).”

It will also ensure the following item, “Set the Fix Version/s” is the 3rd item in the list rather than a first one in a new list.

Reply:
fixed